plugin.video.eafa/changelog.txt

```diff
@@ -1,2 +1,16 @@
+v.1.1.0
```

Comment:
this is no longer used by kodi (only the news tag), so not needed on the submission. Still, you're free to keep it for a global changelog

Reply:
Ah ok, will probably rm in future as it is one less thing to update - figure the commit history is a better log anyway :) Thanks for the heads up.